model_v2/neuroml/ca.nml

```diff
@@ -33,7 +33,7 @@
             <!--<DerivedVariable name="debug" dimension="none" value="shell_volume"/>-->
 
 
-            <DerivedVariable name="effectiveRadius" dimension="length" value="LENGTH_SCALE * sqrt(surfaceArea/(AREA_SCALE * (4 * 3.14159)))"/>  <!-- needed to calc volume of pools, etc. -->
+            <DerivedVariable name="effectiveRadius" dimension="length" value="diam * 0.5"/>  <!-- needed to calc volume of pools, etc. -->
```

Comment:
This only works for approximately spherical segments (like the soma where I'm using it luckily). Would still be better if we had a surfaceArea in nmodl